evalml/tests/component_tests/test_datetime_featurizer.py

```diff
@@ -77,6 +77,6 @@ def test_datetime_featurizer_no_datetime_cols():
 
 def test_datetime_featurizer_numpy_array_input():
     datetime_transformer = DateTimeFeaturizer()
-    X = np.array(['2007-02-03', '2016-06-07', '2020-05-19'], dtype='datetime64')
+    X = np.array([['2007-02-03'], ['2016-06-07'], ['2020-05-19']], dtype='datetime64')
```

Comment:
Previously, we just had code that wrapped this in a `DataFrame`. Now, the conversion code checks for the dimensionality to determine whether to create a Series or DataFrame (since the same conversion code is used for both), so having a 1D numpy would have converted this to a Series instead, which isn't what we want.